### PR TITLE
graphql python guide

### PR DESCRIPTION
**Pull request by raksiv**

### CODE REVIEW

File: docs/guides/graphql.md [hunk 1]
```diff
@@ -394,3 +397,415 @@ To undeploy run the following command:
...
+profiles = collection('profiles').allow('reading','writing')
+```
+
+We can then use the collection within these functions. Each resolver will receive a parent and info arguments, as well as any query or mutation's arguments as keyword arguments.
+
+An example of this is converting the GraphQL query function into Python:
+
+```
+updateProfile(pid: String!, profile: ProfileInput!): Profile
+```
```
Comment (HomelessDinosaur) [inline docs/guides/graphql.md:493]:
```suggestion
```graphql
updateProfile(pid: String!, profile: ProfileInput!): Profile
```
```

File: docs/guides/graphql.md [hunk 2]
```diff
@@ -394,3 +397,415 @@ To undeploy run the following command:
...
+
+Once it starts, the application will be able to receive requests via the API port.
+
+Pressing `ctrl + c` will end the application.
+
+We can use cURL, postman or any other HTTP Client to test our application, however it's better if the client has GraphQL support.
+
+### GraphQL Queries
+
+```text
```
Comment (HomelessDinosaur) [inline docs/guides/graphql.md:614]:
```suggestion
```graphql
```

File: docs/guides/graphql.md [hunk 3]
```diff
@@ -394,3 +397,415 @@ To undeploy run the following command:
...
+    age
+    home_town
+  }
+}
+
+```
+
+And here is the syntax for mutating:
+
+```text
```
Comment (HomelessDinosaur) [inline docs/guides/graphql.md:635]:
```suggestion
```graphql
```

**Review by HomelessDinosaur — CHANGES_REQUESTED** [review]
Just adding graphql for the syntax highlighting. Should be currently supported in our website by Markdoc so no need to make website changes: https://github.com/FormidableLabs/prism-react-renderer/blob/master/src/vendor/prism/includeLangs.js#L22

**Review by HomelessDinosaur — APPROVED** [review]

**Comment by raksiv** [comment]
@HomelessDinosaur I just had to add the nitric start command to the guide.

**Review by HomelessDinosaur — APPROVED** [review]

**Review by HomelessDinosaur — APPROVED** [review]

**Pull request merged**